### ISSUE TITLE
Add encryption

### ISSUE THREAD

**Issue by woheller69** [roozbehzarei/filester#28]
With this library https://github.com/srikanth-lingala/zip4j it should be fairly simple to create encrypted zip files on the fly.

See section **Creating a password protected zip file** in Readme

Maybe an idea for a future enhancement

**Comment by woheller69** [comment]
Exactly, that is the idea. Data in a cloud should never be unencrypted

**Comment by roozbehzarei** [comment]
Thank you for the suggestion. I will definitely add this feature sometime in future (when I find some free time).

> Yes, I think the same. Never unencrypted, in transition and at rest (data).

Since Filester is connecting to the server using HTTPS protocol, all transition is encrypted by default. All that remains is to encrypt the file itself for extra security when it's stored on the cloud.

**Comment by roozbehzarei** [comment]
Currently none of the available public hosting services support encryption. If Filester were to do it by itself, the recipient has to also install Filester for a decryption process which is not the goal.
